tests/components/timer/test_init.py

```diff
@@ -359,3 +404,133 @@ def fake_event_listener(event):
 
     assert results[-1].event_type == EVENT_TIMER_RESTARTED
     assert len(results) == 4
+
+
+async def test_load_from_storage(hass, storage_setup):
+    """Test set up from storage."""
+    assert await storage_setup()
+    state = hass.states.get(f"{DOMAIN}.timer_from_storage")
+    assert state.state == STATUS_IDLE
+    assert state.attributes.get(ATTR_FRIENDLY_NAME) == "timer from storage"
+    assert state.attributes.get(ATTR_EDITABLE)
+
+
+async def test_editable_state_attribute(hass, storage_setup):
+    """Test editable attribute."""
+    assert await storage_setup(config={DOMAIN: {"from_yaml": None}})
+
+    state = hass.states.get(f"{DOMAIN}.{DOMAIN}_from_storage")
+    assert state.state == STATUS_IDLE
+    assert state.attributes.get(ATTR_FRIENDLY_NAME) == "timer from storage"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4821917)